Presentation.Web/app/components/it-contract/tabs/it-contract-tab-main.view.html

```diff
@@ -106,7 +106,16 @@ <h4>{{contract.name}}</h4>
                                            data-ng-disabled="!hasWriteAccess">
                             </select-simple>
                         </div>
-
+                        
+                        <!-- Procurement initiated-->
+                        <div class="form-group">
+                            <label for="procurement-initiated" class="control-label small">Genanskaffelse igangsat</label>
+                            <select-simple id="procurement-initiated"
```

Comment:
this does not work.. it PATCHES when it is loaded. Please use the aproach from DPR
![image](https://user-images.githubusercontent.com/13116889/175240893-931e4b4d-40f3-4c9d-824f-44171a266bb2.png)

controller code:
![image](https://user-images.githubusercontent.com/13116889/175240958-cdc09265-2891-4edd-86d0-95896b3180d3.png)

![image](https://user-images.githubusercontent.com/13116889/175241047-84e08558-733d-4056-8d8f-6d46394becf4.png)